tests/test_data/message/thread_technical.json

```diff
@@ -0,0 +1,66 @@
+{
```

Comment:
Even if there is nothing to reuse/extend, these 2 JSON files can be minimised, are you purely changing the category?

Reply:
As above.